blackjack/blackjack_test.go

```diff
@@ -112,83 +136,84 @@ func TestPlayerBust(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	g.Player.Action = blackjack.ActionHit
+	players := []blackjack.Player{
+		{Name: "Planty"},
+		{Name: "Kevin"},
+		{Name: "Donald"},
+	}
+
+	g.Players = players
+
+	g.Players[0].Action = blackjack.ActionStand
```

Comment:
Perhaps it would be clearer to set up these test players as literals?

```go
g.AddPlayer(blackjack.Player{
	Name: "Planty",
	Action: blackjack.ActionStand,
})
g.AddPlayer...
```

Reply:
(It certainly doesn't feel like we should be reaching inside someone else's slice and messing with it. That's just rude.)